index.bs

```diff
@@ -765,12 +770,12 @@ that are returned to the caller when a new credential is created, or a new asser
         "{{Credential/[[discovery]]/remote}}".
 
     :   <dfn>\[[identifier]]</dfn>
-    ::  This [=internal slot=] contains the [=credential ID=], chosen by the platform with help from the authenticator.
+    ::  This [=internal slot=] contains the [=credential ID=], chosen by the authenticator.
         The [=credential ID=] is used to look up credentials for use, and is therefore expected to be globally unique
         with high probability across all credentials of the same type, across all authenticators.
 
         Note: This API does not constrain
-        the format or length of this identifier, except that it MUST be sufficient for the platform to uniquely select a key.
+        the format or length of this identifier, except that it MUST be sufficient for the [=authenticator=] to uniquely select a key.
         For example, an authenticator without on-board storage may create identifiers containing a [=credential private key=]
         wrapped with a symmetric key that is burned into the authenticator.
```

Comment:
while we are here: 
that is burned -> , where the latter key is burned

Reply:
Or how about s/key that is burned/key burned/ ?